edi_account_oca/models/res_partner.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    edi_account_move_exchange_type_id = fields.Many2one(
+        string="EDI Exchange Type For Invoices",
```

Comment:
```suggestion
        string="EDI Exchange type for outgoing invoices",
```

Reply:
You are not using this configuration anywhere, at least you need to add someting on the Connector :thinking:

Reply:
What do you mean? It has to be documented... not used.
The goal is to be able to decide which type to use to send out an invoice for that specific customer.

Reply:
Which is the goal then? adding a field to use it somewhere else? Wouldn't have more sense to add the connector script?

Reply:
@etobella ping :)